.hlint.yaml

```diff
@@ -1,17 +1,13 @@
-- arguments:
-  - -XTypeApplications # TypeApplications are disabled by default in HLint,
-                       # although the documentation says otherwise
```

Comment:
Was it fixed?

Reply:
I don't know. But in any case, the `hlint` config for Universum users is no place for custom extensions.